examples/amp-story/analytics.html

```diff
@@ -63,7 +63,10 @@
               },
               "trackPageview": {
                 "on": "story-page-visible",
-                "request": "pageView"
+                "request": "pageView",
+                "storySpec": {
+                  "repeat": false
```

Comment:
Just recall one thing. There was a request to send out a request for the number of times specified for other triggers. For example, one can specify `"repeat" : 3`. And only the first three such requests will be sent out. 
I'll leave it to you to decide if we will ever support that use case : )

Reply:
That'd be great. Let me see if I can find the GH issue.

Reply:
#12298

Reply:
If that already exists, would be great to leverage The existing configuration!

Reply:
we haven't get the chance to build that yet 😢

Reply:
Do you mind if we do this as a follow up, @zhouyx ? This PR has already gotten quite a bit of turnaround, so I think it would be better to address separately.

Reply:
Friendly ping @zhouyx

Reply:
oops I'm very sorry, didn't mean to block the PR. That sounds good to me. Even we adopt that it would need some redesign to have `repeat: 1` works in this case.